chapters/en/chapter10/9.mdx

```diff
@@ -0,0 +1,234 @@
+<!-- DISABLE-FRONTMATTER-SECTIONS -->
+
+# End-of-chapter quiz
```

Comment:
This is just a placeholder - I'll update it once the rest of the sections are written